src/sdk/ConversationTranslator/Conversation.ts

```diff
@@ -138,6 +180,30 @@ export class ConversationImpl extends Conversation implements IDisposable {
         return this.privLanguage;
     }
 
+    public get isMutedByHost(): boolean {
+        return this.privParticipants.me?.isHost ? false : this.privParticipants.me?.isMuted;
+    }
+
+    public get isConnected(): boolean {
+        return this.privIsConnected && this.privIsReady;
+    }
+
+    public get participants(): Participant[] {
+        return this.toParticipants(true);
+    }
+
+    public get me(): Participant {
+        return this.toParticipant(this.privParticipants.me);
+    }
+
+    public get host(): Participant {
+        return this.toParticipant(this.privParticipants.host);
+    }
+
+    /**
+     * Create a conversation impl
+     * @param speechConfig
+     */
     public constructor(speechConfig: SpeechTranslationConfig) {
```

Comment:
The constructor should be private for consistency with the C/C++/C# code.

Reply:
The code won't compile if this change is made.